taskcluster/ci/config.yml

```diff
@@ -0,0 +1,48 @@
+---
+trust-domain: scriptworker
+
+task-priority: high
+
+taskgraph:
+    register: scriptworker_taskgraph:register
+    repositories:
+        scriptworker:
+            name: "Scriptworker-scripts"
+            project-regex: scriptworker-scripts$
+            default-repository: https://github.com/mozilla-releng/scriptworker-scripts
+            default-ref: master
+            type: git
+
+workers:
+    aliases:
+        b-linux:
+            provisioner: scriptworker-{level}
+            implementation: docker-worker
+            os: linux
+            worker-type: 'b-linux'
+        images:
+            provisioner: scriptworker-{level}
+            implementation: docker-worker
+            os: linux
+            worker-type: 'images'
+
+docker:
+    email: "release+dockerhub+services@mozilla.com"
+    user: mozillarelengservices
+
+notify:
+    email:
+        # TODO by-project/level
+        - aki+scriptworker-scripts@mozilla.com
```

Comment:
Nit: do we want to update this now or in a follow-up, once we stabilize the CI via taskgraph?

Reply:
Let's do it now :)